### ISSUE TITLE
Failed building wheel for aiohttp==3.4.3

### ISSUE THREAD

**Issue by oleksandr-kuzmenko** [aio-libs/aiohttp#3241]
OS - ubuntu bionic
python - 3.7
aiohttp - 3.4.3

no problems with aiohttp 3.4.2

```
----------------------------------------
  Failed building wheel for aiohttp
  Running setup.py clean for aiohttp
Failed to build aiohttp
Installing collected packages: aiohttp, aiokafka, asyncpg, asyncpgsa, cantal, cbor, graphene, graphql-core, graphql-server-core, kafka-python, rororo, schedule, sqlalchemy, trafaret, trafaret-config, uvloop, async-shim, concierge-aiohttp, conciergelib, logevo, async-timeout, attrs, chardet, idna, multidict, yarl, PyYAML, rx, aniso8601, graphql-relay, promise, six
  Running setup.py install for aiohttp ... error
    Complete output from command /usr/bin/python3.7 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-pmcpipif/aiohttp/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-1268nolh/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.7
    creating build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_routedef.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/tracing.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/test_utils.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/client_proto.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_runner.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/tcp_helpers.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/cookiejar.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/locks.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/payload.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/connector.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_exceptions.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_middlewares.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_protocol.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/hdrs.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/http_exceptions.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_urldispatcher.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/__init__.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/http.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/http_writer.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/typedefs.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/client_ws.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/worker.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/signals.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/client_reqrep.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_app.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/pytest_plugin.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/abc.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/streams.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/http_websocket.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/client.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/log.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/base_protocol.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_server.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/helpers.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/payload_streamer.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_fileresponse.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/client_exceptions.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_response.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/multipart.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/http_parser.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/formdata.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_ws.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/web_request.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/frozenlist.py -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/resolver.py -> build/lib.linux-x86_64-3.7/aiohttp
    running egg_info
    writing aiohttp.egg-info/PKG-INFO
    writing dependency_links to aiohttp.egg-info/dependency_links.txt
    writing requirements to aiohttp.egg-info/requires.txt
    writing top-level names to aiohttp.egg-info/top_level.txt
    reading manifest file 'aiohttp.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    warning: no files found matching 'aiohttp' anywhere in distribution
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
    warning: no previously-included files matching '*.pyd' found anywhere in distribution
    warning: no previously-included files matching '*.so' found anywhere in distribution
    warning: no previously-included files matching '*.lib' found anywhere in distribution
    warning: no previously-included files matching '*.dll' found anywhere in distribution
    warning: no previously-included files matching '*.a' found anywhere in distribution
    warning: no previously-included files matching '*.obj' found anywhere in distribution
    warning: no previously-included files found matching 'aiohttp/*.html'
    no previously-included directories found matching 'docs/_build'
    writing manifest file 'aiohttp.egg-info/SOURCES.txt'
    copying aiohttp/_cparser.pxd -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_find_header.c -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_find_header.h -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_find_header.pxd -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_frozenlist.c -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_frozenlist.pyx -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_headers.pxi -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_helpers.c -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_helpers.pyi -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_helpers.pyx -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_http_parser.c -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_http_parser.pyx -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_http_writer.c -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_http_writer.pyx -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_websocket.c -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/_websocket.pyx -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/frozenlist.pyi -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/py.typed -> build/lib.linux-x86_64-3.7/aiohttp
    copying aiohttp/signals.pyi -> build/lib.linux-x86_64-3.7/aiohttp
    running build_ext
    building 'aiohttp._websocket' extension
    creating build/temp.linux-x86_64-3.7
    creating build/temp.linux-x86_64-3.7/aiohttp
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fdebug-prefix-map=/build/python3.7-gdrLxR/python3.7-3.7.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.7m -c aiohttp/_websocket.c -o build/temp.linux-x86_64-3.7/aiohttp/_websocket.o
    aiohttp/_websocket.c:17:10: fatal error: Python.h: No such file or directory
     #include "Python.h"
              ^~~~~~~~~~
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    
    ----------------------------------------
```

**Comment by webknjaz** [comment]
@asvetlov wasn't it supposed to fall back to pure-python implementation as per your recent PR?

**Comment by asvetlov** [comment]
Uups. Forgot to return `CCompilerError` handling back

**Comment by asvetlov** [comment]
Fixed by fd335f2d
